accepted/future-releases/primary-constructors/feature-specification.md

```diff
@@ -874,8 +892,9 @@ reason for this error is that the modifier `covariant` must be specified on
 the declaration of `v` which is known to exist, not on the parameter.*
 
 A compile-time error occurs if _p_ has both of the modifiers `covariant`
-and `final`. *A final instance variable cannot be covariant, because being
-covariant is a property of the setter.*
+and `final`, also if the latter is implicitly induced. *A final instance
```

Comment:
Just trying to understand this better. Could you give me an example of this change? "if the latter is implicitly induced".